src/lib/libmints/molecule.cc

```diff
@@ -245,6 +234,18 @@ Molecule& Molecule::operator=(const Molecule& other)
         atoms_.push_back((*iter)->clone(atoms_, geometry_variables_));
     }
 
+
+    // These are symmetry related variables, and are filled in by the following functions
+    pg_             = boost::shared_ptr<PointGroup>(new PointGroup(other.pg_));
```

Comment:
I recall there is a reason we do not copy the PG. @loriab Do you remember what this is?